values.yaml

```diff
@@ -3,7 +3,7 @@ podSecurityContext: {}
 securityContext: {}
 
 common-image-stuff: &common-image-stuff
-  tag: '1.5.2'
+  tag: '1.7-RC'
```

Comment:
it would be good to have a true release of 1.7 before merging this PR

Reply:
Moved to 1.7-RC since we have people interested to test the RC.  @groldan any comment/thoughts?

Reply:
@danduk82 the point of an RC is to be battle tested and either become the final release or fix some urgent bug and then release. I don't  know how do you handle that in the helm chart, but to that effect we need a helm chart release.
Chances are, create a 1.7 branch and merge there, then tag it so interested parties can test it?
On another note, wouldn't it be easier to follow if the helm chart version followed the gscloud version?